src/main/java/xyz/kyngs/herbot/util/MemberUtil.java

```diff
@@ -0,0 +1,28 @@
+package xyz.kyngs.herbot.util;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.User;
+
+public class MemberUtil {
+
+    public static User getUser(String name, JDA jda) {
+        try {
+            User byID = jda.getUserById(name);
+            if (byID != null) return byID;
+        } catch (Exception ignored) {
+        }
+        try {
+            User byTag = jda.getUserByTag(name);
+            if (byTag != null) return byTag;
+        } catch (Exception ignored) {
```

Comment:
Why do you want to ignore these exceptions?

Reply:
So if jda.getUserByID throws an exception it indicates that the supplied id was not an id but some other text, for example "OwO", therefore we want to ignore the exception and check if the test is not infact some name or tag.